working/0546-patterns/patterns-feature-specification.md

```diff
@@ -1940,6 +1968,15 @@ To match a pattern `p` against a value `v`:
         method of the same name on the matched value, with `c` as the argument
         returns true.
 
+*   **Cast**:
+
+    1.  If the runtime type of `v` is not a subtype of the cast type of `p` then
+        throw a runtime exception. *Note that we throw even if this appears in a
```

Comment:
exception -> error. (It'll be a `TypeError`)

Reply:
Done.